pkg/controller/telemetry/map_metric.go

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/cilium/ebpf"
+)
+
+const (
+	mapMetricFlushInterval = 15 * time.Second
+)
+
+type MapMetricController struct {
+}
+
+type mapEntrycountMetric struct {
+	mapName    string
+	entryCount uint32
+}
+
+type mapMetricLabels struct {
+	mapName  string
+	nodeName string
+}
+
+type totalMapMetricLabels struct {
+	nodeName string
```

Comment:
can you remove this sturct, so similar with mapMetricLabels

Reply:
Considering code reuse and style before, I think there is no need to delete it

Reply:
I doubt isn't this duplicate with mapMetricLabels

Reply:
One is a label for a single map, and the other is a label for the total number of maps, requiring different struct to label different data